frontend/src/metabase/admin/settings/components/widgets/FormattingWidget.jsx

```diff
@@ -18,7 +18,6 @@ const SETTING_TYPES = [
     ],
     column: {
       semantic_type: TYPE.Temporal,
-      unit: "second",
```

Comment:
this unit being set was hiding the time settings once we were obeying the hidden column settings